presto-docs/src/main/sphinx/release/release-0.266.rst

```diff
@@ -0,0 +1,39 @@
+=============
+Release 0.266
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix resource group concurrency for multi coordinator.
```

Comment:
Little bit unclear as to what is the user impact here. 
Maybe reword it on the lines of something like
Fix resource group concurrency for multi coordinator to reduce queuing time/ to address improper queuing logic

Reply:
Let's use this instead:
`Fix resource group concurrency for multi coordinator to not allow more than configured queries to run.`